src/network/networkd-route.c

```diff
@@ -749,6 +750,17 @@ int network_add_default_route_on_device(Network *network) {
         return 0;
 }
 
+static const char * const route_type_table[__RTN_MAX] = {
+        [RTN_UNICAST]     = "unicast",
+        [RTN_BLACKHOLE]   = "blackhole",
+        [RTN_UNREACHABLE] = "unreachable",
+        [RTN_PROHIBIT]    = "prohibit",
+        [RTN_THROW]       = "throw",
+};
+
+assert_cc(__RTN_MAX <= UINT8_MAX);
```

Comment:
shouldn't this rather check for UCHAR_MAX?

Reply:
Fixed. Thanks.